en/micro-integrator/docs/references/connectors/servicenow-connector/settingup-servicenow-instance.md

```diff
@@ -0,0 +1,24 @@
+# Setting up the ServiceNow Instance
+
+The ServiceNow connector allows you to access the ServiceNow REST API of through WSO2 EI. To use the REST API, we need to have a [Servicenow](https://www.servicenow.com/) account. 
+
+## Signing Up for Servicenow
```

Comment:
There is some inconsistency here. Shall we call it "ServiceNow" everywhere?

Reply:
Done